hyperjumptech/monika
# Add redis url support

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Implement $1008

## How did you implement / how did you fix it  
1. Adding uri to redis prober
2. Adding redis uri validation
3. Adding to documentation

## how to test  
1. Use the following yaml
```yaml
probes:
   - id: 'redis-1'
    name: 'redis-test'
    redis:
      - uri: 'redis://172.17.0.1:6379'

notifications:
  - id: desktop
    type: desktop
```
 2. Run monika with `-c config.yaml` above
 3. Expect to see redis pinged in the logs


## before PR
![image](https://user-images.githubusercontent.com/964106/236087482-27483346-5d85-4a31-90eb-7882ba57b350.png)

## after PR
![image](https://user-images.githubusercontent.com/964106/236685866-ffacd13f-d101-4883-a47d-d365a887adf1.png)
